static/sw.js

```diff
@@ -0,0 +1,10 @@
+importScripts('/bower_components/sw-toolbox/sw-toolbox.js');
+
+toolbox.options.debug = true;
+toolbox.router.default = toolbox.fastest;
+
+const filesToCache = [
+    '/'
```

Comment:
Precaching all samples only makes sense if they work offline as well. This isn't the case for all samples. Some require amp components which might not be in the browser cache yet. Other require external components which can't be cached. We first need to implement a proper offline strategy for ABE first. (see #225 for more about this).